docs/release.rst

```diff
@@ -115,30 +115,13 @@ Upcoming Fedora release & package build
 Creating and anaconda release and build for an upcoming Fedora release is pretty similar to a Rawhide build
 with a few key differences:
 
-- the branches are named differently
+- the upstream project branch is named f<version>-release
```

Comment:
It doesn't matter now, but do we want to keep the `-release` suffix in the future?

Reply:
Please remove the `-release` suffix. No reason to have it.

Reply:
Hmm, that's a good point - with just a single branch we can just name it f<version> - the only issue being where it possibly hardcoded & previous releases that have it, including F37.

Reply:
Also the branch name will be the same asin Fedora distgit, so changing some of the references to "f<version> upstream branch" to note the difference.

Reply:
Or it could be `fedora-38` to make it consistent with `rhel-9`.

Reply:
`fedora-NN` seems a good "new style" to me.

Reply:
Agreed - `fedora-NN` it will be! :)